### DISCUSSION TITLE
Как натравить Zapret на OpenVPN TCP?

### DISCUSSION THREAD

**Discussion by Gleb-pro-admin** [bol-van/zapret#877] [Q&A]
GoodbyeDPI имеет [ветку](https://github.com/ValdikSS/GoodbyeDPI/tree/openvpn) с поддержкой определения OpenVPN. Возможно ли использовать `nfqws` для обхода ограничений `OpenVPN TCP`, осуществляемых с помощью DPI? Какие настройки потребуются? Есть ли опробованные стратегии?

**Comment by bol-van** [comment]
В режиме desync any protocol можно насиловать любые протоколы.
Все необходимое описано в доке.
